src/_imagingcms.c

```diff
@@ -390,7 +389,7 @@ _buildTransform(
 
     Py_END_ALLOW_THREADS
 
-    if (!hTransform) {
+        if (!hTransform) {
         PyErr_SetString(PyExc_ValueError, "cannot build transform");
     }
```

Comment:
This doesn't look right. It's caused by the lack of semicolon after `Py_END_ALLOW_THREADS`.